### ISSUE TITLE
Remmina echo wrong or unclear error for failed ssh connection to server with disabled password mechanism

### ISSUE THREAD

**Issue by ssten** [FreeRDP/Remmina#1372]
My settings are:
* Client (OS name and version): Gentoo GNU/Linux amd64 profile
* Remmina version (remmina --version): net-misc/remmina (1.2.0_rc24@07.11.2017)
Remmina - Version 1.2.0-rcgit-24 (git rcgit-24)
* Desktop environment (GNOME, Unity, KDE, ..): XFce 4.12
* Connecting to (OS and version): CentOS Linux release 7.2.1511 (Core) 

sshd on server is configured to accept **only** public keys:
> $ ssh 192.168.11.13
> Permission denied (publickey).

Trying to connect to this server with remmina, configured to use password authentication I got the following error message:
![ssh_password-no](https://user-images.githubusercontent.com/2209401/32833738-f2a668b2-ca10-11e7-94b8-ba620c73e87f.png)
But it can appear not only when password mech is disabled, but also when password type limit is gone.

I suggest to add in connection list columm with icon of last connection status.
And in connection details — tab with log messages for last session.

**Comment by antenore** [comment]
Yes, you are right, but I didn't get your suggestion, can you explain more please?

Errors are not well managed in the SSH part, I'll work on it while refactoring Remmina.

**Comment by ssten** [comment]
First of all, generally I suggest to add a tab with last connection messages and field to type a debug options like *-v*, *-vv* and *-vvv* in openssh client.

The next suggestion is about algorythm: prior to try selected auth mech, remmina should enshure, this mechanism in supported by server.
If connection is confugured to use unsupported authentication mechsnism it should display proper error message.
Not «password authentication failed», but «server is configured to **not** use password authentication».

**Comment by antenore** [comment]
Some notes

1) Debug
You have a Debug window accessible from the menu and you can select the debug level in the preferences
2) UI
In the future we will work again on the interface and I'll keep in mind your advice.

**Comment by antenore** [comment]
WARNING:
The Remmina project is moving to gitlab.com, in order to keep receiving notification about this issue,
you should have a gitlab account and link it to github.
The migration will happen in the next days.
Thanks for your support and understanding.

cc @antenore @ssten
